keras_retinanet/utils/transform.py

```diff
@@ -0,0 +1,241 @@
+import numpy as np
+
+DEFAULT_PRNG = np.random
+
+
+def colvec(*args):
+    """ Create a numpy array representing a column vector. """
+    return np.array([args]).T
+
+
+def transform_aabb(transform, aabb):
+    """ Apply a transformation to an axis aligned bounding box.
+
+    The result is a new AABB in the same coordinate system as the original AABB.
+    The new AABB contains all corner points of the original AABB after applying the given transformation.
+
+    # Arguments
+        transform: The transormation to apply.
+        x1:        The minimum X value of the AABB.
+        y1:        The minimum y value of the AABB.
+        x2:        The maximum X value of the AABB.
+        y2:        The maximum y value of the AABB.
+    # Returns
+        The new AABB as tuple (x1, y1, x2, y2)
+    """
+    x1, y1, x2, y2 = aabb
+    # Transform all 4 corners of the AABB.
+    points = transform.dot([
+        [x1, x2, x1, x2],
+        [y1, y2, y2, y1],
+        [1,  1,  1,  1 ],
+    ])
+
+    # Extract the min and max corners again.
+    min_corner = points.min(axis=1)
+    max_corner = points.max(axis=1)
+
+    return [min_corner[0], min_corner[1], max_corner[0], max_corner[1]]
+
+
+def _random_vector(min, max, prng = DEFAULT_PRNG):
```

Comment:
Non-aligned default arguments are written down without spaces around the equal sign (same for some other places).

Reply:
pep8 could be used for catching those spaces

Reply:
One problem with using `pytest --pep8` for that is that we do use spaces around the parameter assignment on multi-line function calls for alignment. The pep8 test doesn't distinguish between that and this type of whitespace.

Reply:
Anyway, fixed, I think.